src/parsing.rs

```diff
@@ -24,6 +24,7 @@ enum ControlOperator {
 
 pub const SCOPE_MARKER: &str = "_CDDL_CODEGEN_SCOPE_MARKER_";
 pub const EXTERN_MARKER: &str = "_CDDL_CODEGEN_EXTERN_TYPE_";
+pub const RAW_BYTES_MARKER: &str = "_CDDL_CODEGEN_RAW_BYTES_TYPE_";
```

Comment:
missing test?

Reply:
I didn't add one since it depended a lot on CML stuff. I suppose I could extract the CML trait + write some dummy object to implement it and have the test pull in those files.

Reply:
I ended up just adding tests for it, but I had to add an extra variant to the standard error type that we had in CML. Although it's a variant that could be useful in general to other non-CML people using it.